panels/site_sections/panel_app_management.py

```diff
@@ -231,3 +231,26 @@ def everything(self, out, session):
                 ' / '.join(app.tech_needs_labels) + (' / ' if app.other_tech_needs else '') + app.other_tech_needs,
                 app.applied.strftime('%Y-%m-%d')
             ] + panelists)
+
+    def staff_schedule(self, session, attendee_id):
+        attendee = session.attendee(attendee_id)
+        events = defaultdict(lambda: defaultdict(lambda: (1, '')))
+        for pa in attendee.panel_applications:
+            if pa.event is not None:
+                for timeslot in pa.event.half_hours:
+                    rowspan = pa.event.duration if timeslot == pa.event.start_time else 0
+                    events[timeslot][pa.event.location_label] = (rowspan, pa.event.name)
+
+        schedule = []
+        locations = sorted(set(sum([list(locations) for locations in events.values()], [])))
+        if events:
```

Comment:
I don't see anywhere where events could be anything except the dict you defined above. And that dict also doesn't make any sense (this might be me). What is supposed to be happening here?